sympy/physics/wigner.py

```diff
@@ -230,6 +232,10 @@ def wigner_3j(j_1, j_2, j_3, m_1, m_2, m_3):
         return S.Zero
     if (abs(m_1) > j_1) or (abs(m_2) > j_2) or (abs(m_3) > j_3):
         return S.Zero
+    if int(j_1 - m_1) != (j_1 - m_1) or \
+            int(j_2 - m_2) != (j_2 - m_2) or \
+            int(j_3 - m_3) != (j_3 - m_3):
+        return S.Zero
```

Comment:
There is an `int_valued` function in `sympy.core.numbers` that could be used here.

Reply:
OK, it's updated.